tests/models/test_cli.py

```diff
@@ -75,54 +73,6 @@ def sk_model(iris_data):
     return knn_model
 
 
-def test_predict_with_old_mlflow_in_conda_and_with_orient_records(iris_data):
```

Comment:
This test has been disabled since 1.0. The functionality here is covered by `test_predict()` in the module, so deleting it instead of duplicating a test after reenabling.